tests/kafkatest/tests/benchmark_test.py

```diff
@@ -26,16 +28,22 @@
 DEFAULT_RECORD_SIZE = 100  # bytes
 
 
-class Benchmark(KafkaTest):
+class Benchmark(Test):
```

Comment:
This was changed because Kafka is now started in these tests using the security protocol specified in the test - hence the creation of the Kafka service can no longer be in the constructor. Since the current KafkaTest is suitable for tests which dont set security protocol, it seemed reasonable to leave it as is.